frontend/apps/reader/modules/readeruserhyph.lua

```diff
@@ -210,13 +304,13 @@ function ReaderUserHyph:modifyUserEntry(word)
 
                         if self:checkHyphenation(new_suggestion, word) then
                             -- don't save if no changes
-                            if Utf8Proc.lowercase(new_suggestion) ~= input_dialog.old_hyph_lowercase then
+                            if Utf8Proc.lowercase(new_suggestion, NORM) ~= input_dialog.old_hyph_lowercase then
                                 self:updateDictionary(word, new_suggestion)
                             end
                             UIManager:close(input_dialog)
                         else
                             UIManager:show(InfoMessage:new{
-                                text = T(_("Invalid hyphenation!"), self.dict_file),
+                                text = _("Invalid hyphenation!"),
```

Comment:
Ignoring for a second that it wasn't used, that does look like useful info to stick on? But in that case more like:

```suggestion
                                text = _("Invalid hyphenation!") .. "\n" .. self.dict_file,
```

(Keep in mind I'm only skimming the diff though.)

Reply:
No, thats absolutely not necessary. Maybe something like
```
                           UIManager:show(InfoMessage:new{
                                text = T(_("Invalid hyphenation!\n%1\ndoes not match\n%2"),
                                    "    " .. word, "    " .. input_dialog:getInputText())
                            })
````
would give the user a more useful hint.

Reply:
This will look like:
![grafik](https://user-images.githubusercontent.com/36999612/158679471-ca7657f8-063c-4b1e-ad89-495ec6a9ca0b.png)

Reply:
If the original input is normally visible as in your screenshot I don't think that's necessary.

Reply:
Yes it is! -> So from my side it is ready to merge.